gui/src/pages/AddNewModel/ConfigureProvider.tsx

```diff
@@ -301,7 +368,36 @@ function ConfigureProvider() {
               })}
             </details>
           )}
-
+          {providerName === "openrouter" && (
+            <div className="mb-2">
+              <label htmlFor="model">Select a Model</label>
+              <Select
+                id="model"
+                className="border-2 border-gray-200 rounded-md p-2 m-2 w-full"
+                {...formMethods.register("model", { required: true })}
+              >
+                <option value="">Select a model</option>
+                {providers.openrouter?.packages.map((pkg) => (
+                  <option key={pkg.params.model} value={pkg.params.model}>
+                    {pkg.title}
+                  </option>
+                ))}
+              </Select>
+            </div>
+          )}
+          {providerName === "openrouter" && (
+            <button
```

Comment:
this button styling is a bit non-conform, can you match it to other buttons

Reply:
Yes. I will check it out. I didn't see a good example in the current provider code. I'll do a little more research.

Reply:
Done